test-clients/src/main/java/com/hedera/services/yahcli/Yahcli.java

```diff
@@ -85,6 +78,12 @@ public class Yahcli implements Callable<Integer> {
             paramLabel = "payer")
     String payer;
 
+    @Option(
+            names = {"-s", "--schedule"},
+            paramLabel = "schedule",
+            description = "boolean schedule param, if interested to schedule for signatures")
```

Comment:
```suggestion
            description = "true if the transaction should be scheduled, false otherwise")
```
Is this boolean to schedule any transaction?

Reply:
yes

Reply:
but as of right now it is enabled just on send and update